pom.xml

```diff
@@ -276,6 +277,11 @@
 	<!-- Dependencies common to all child projects -->
 	<dependencies>
 		<!-- Good overview about annotation alternatives: http://stackoverflow.com/questions/4963300/which-notnull-java-annotation-should-i-use -->
+		<dependency>
```

Comment:
This dependency should be added to the module `kb-importer` only. When added like this, all other child modules will have this dependency even though they do not need it.